_includes/head.html

```diff
@@ -0,0 +1,12 @@
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  {% seo %}
+  <link rel="stylesheet" href="{{ "/assets/main.css" | relative_url }}">
+  <link rel="shortcut icon" type="image/png" href="assets/favicon.png">
+  {% feed_meta %}
+  {% if jekyll.environment == 'production' and site.google_analytics %}
+    {% include google-analytics.html %}
+  {% endif %}
```

Comment:
Is Google Analytics necessary?

Reply:
Well, it's not configured at all so it wouldn't be on. But I highly recommend that we do configure at some point, anonymized analytics are awesome